list.rb

```diff
@@ -0,0 +1,18 @@
+require './my_enumerable'
```

Comment:
Kindly use  require_relative to import MyEnumerable since using require_relative for files in your directory instead of require is a ruby convention.

Reply:
- [ ] Kindy ensure all the test cases pass I found 2 of them failing ` list.all? {|e| e < 5} return false` instead of true and `list.any? {|e| e == 5} return true ` instead of false